src/styles/themes/default.js

```diff
@@ -69,6 +69,10 @@ const darkTheme = {
     reactionActive: '#003C73',
     badgeAdHoc: colors.pink600,
     badgeAdHocHover: colors.pink700,
+    mentionText: colors.blue200,
+    mentionBG: colors.blue700,
+    ourMentionText: colors.green200,
+    ourMentionBG: colors.green700,
```

Comment:
In the design doc we said these should be: 
```
mentionText: colors.blue100
mentionBG: colors.blue600
ourMentionText: colors.green100
ourMentionBG: colors.green600
```
In case those aren't defined: 

- blue100: #BOD9FF
- blue600: #0164BF
- green100: #B1F2D6
- green600: #008C59